server/base/src/main/java/org/apache/accumulo/server/fs/VolumeUtil.java

```diff
@@ -89,7 +89,7 @@ public static Path switchVolume(String path, FileType ft, List<Pair<Path,Path>>
     return null;
   }
 
-  private static LogEntry switchVolumes(LogEntry le, List<Pair<Path,Path>> replacements) {
+  public static LogEntry switchVolumes(LogEntry le, List<Pair<Path,Path>> replacements) {
```

Comment:
You may want to add the visible for testing to document the reason this is public.

Reply:
I made the method protected instead.